packages/back-end/types/webhook.d.ts

```diff
@@ -11,14 +11,22 @@ export interface WebhookInterface {
   error: string;
   created: Date;
   useSdkMode: boolean;
-  sendPayload: boolean;
+  /** @deprecated */
+  sendPayload?: boolean;
+  payloadFormat?: WebhookPayloadFormat;
   sdks: string[];
   headers?: string;
   httpMethod?: WebhookMethod;
 }
 
 export type WebhookMethod = "GET" | "PUT" | "POST" | "DELETE" | "PURGE";
 
+export type WebhookPayloadFormat =
+  | "standard"
+  | "standard-no-payload"
+  | "sdkPayload"
+  | "none";
```

Comment:
It would be nice to derive this from a `const` array from the model so we have one single source of truth for all these values.

Reply:
Going to leave this alone for now. I find the plain types much more readable, plus its in line with how we generally do it.